lib/tapioca.rb

```diff
@@ -15,6 +22,25 @@ def self.silence_warnings(&blk)
   end
 
   class Error < StandardError; end
+
+  SORBET_PATH = T.let("sorbet", String)
+  SORBET_CONFIG = T.let("#{SORBET_PATH}/config", String)
+  TAPIOCA_PATH = T.let("#{SORBET_PATH}/tapioca", String)
+  TAPIOCA_CONFIG = T.let("#{TAPIOCA_PATH}/config.yml", String)
+
+  DEFAULT_COMMAND = T.let("bin/tapioca", String)
+  DEFAULT_POSTREQUIRE = T.let("#{TAPIOCA_PATH}/require.rb", String)
+  DEFAULT_RBIDIR = T.let("#{SORBET_PATH}/rbi", String)
```

Comment:
We use `SORBET_PATH` a few line before. Should we rename it to `SORBET_DIR` or rename this one `DEFAULT_RBI_PATH`?

Also, let's add an underscore:
```suggestion
  DEFAULT_RBI_DIR = T.let("#{SORBET_PATH}/rbi", String)
```